kusto/ingest/internal/filesystem/filesystem_test.go

```diff
@@ -40,9 +40,9 @@ func TestFormatDiscovery(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		got := FormatDiscovery(test.input)
+		got := properties.DataFormatDiscovery(test.input)
 		if got != test.want {
-			t.Errorf("TestFormatDiscovery(%s): got %s, want %s", test.input, got, test.want)
+			t.Errorf("TestFormatDiscovery(%s): got '%s', want '%s'", test.input, got, test.want)
```

Comment:
Don't add the quotes.

If you absolutely thing you need them, use %q.

Reply:
Ok.
I want to make sure that if by any change we get an empty string, it will be visible in the error